modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -313,7 +313,11 @@ export default class Tileset3D {
     this.frameStateData = {};
     this.lastUpdatedVieports = null;
 
-    this._queryParams = {};
+    if (this.loadOptions['3d-tiles'] && 'queryParams' in this.loadOptions['3d-tiles']) {
```

Comment:
I suggest initializing `3d-tiles` specific data in `_initializeTiles3DTileset` method.